.gitignore

```diff
@@ -228,3 +228,4 @@ tags
 [Ll]ocal
 pyvenv.cfg
 pip-selfcheck.json
+poetry.toml
```

Comment:
What is `poetry.toml`?

Reply:
poetry.toml is the file that is created by running `poetry config --local virtualenvs.in-project true`, which is in the readme.